lib/internal/Magento/Framework/App/CacheInterface.php

```diff
@@ -22,10 +24,10 @@ interface CacheInterface
     public function getFrontend();
 
     /**
-     * Load data from cache by id
+     * Load Data from Cache by ID
      *
-     * @param  string $identifier
-     * @return string
+     * @param string $identifier
+     * @return bool|string
```

Comment:
Probably adding bool here is not the right approach, as far as I could see, currently false is being returned in case of cache not available, in this case, we likely should throw an exception instead.

I believe the right solution here is to improve/change implementation instead of changing annotation.

Reply:
From @gabrieldagama in https://github.com/magento/magento2/pull/31987#pullrequestreview-629801370:
> After discussions, it seems that the return bool is actually by design since `FrontendCache` is encapsulated inside the `CacheInterface` and its contract does return bool.
>
> I believe in this case it is likely that the return type is missing, so we should update it.